### PR TITLE
[Document][202012] Supply the missing ingress/egress port profile list in document

### PR DESCRIPTION
**Pull request by stephenxs**
**What I did**

Supply the missing ingress/egress port profile list tables in the document

Signed-off-by: Stephen Sun <stephens@nvidia.com>

**Why I did it**

**How I verified it**

**Details if related**

### CODE REVIEW

**Review by keboliu — APPROVED** [review]

File: doc/Configuration.md [hunk 1]
```diff
@@ -503,6 +505,43 @@ This kind of profiles will be handled by buffer manager and won't be applied to
  
 ```
 
+### Buffer port ingress profile list
+
+```
+{
+"BUFFER_PORT_[BUFFER_PROFILE|INGRESS_PROFILE_LIST": {
```
Comment (neethajohn) [inline doc/Configuration.md:512]: This needs to be fixed

**Review by neethajohn — APPROVED** [review]

**Pull request merged**